### PR TITLE
LPD-19858 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://issues.liferay.com/browse/LPD-19858

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

File: modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java [hunk 1]
```diff
@@ -176,6 +181,22 @@ private boolean _isAnalyticsTrackingEnabled(
...
 	}
 
+	private boolean _isCookiesExplicitConsentMode(ThemeDisplay themeDisplay) {
+		try {
+			CookiesPreferenceHandlingConfiguration
+				cookiesPreferenceHandlingConfiguration =
+					_cookiesConfigurationProvider.
+						getCookiesPreferenceHandlingConfiguration(themeDisplay);
+
+			return cookiesPreferenceHandlingConfiguration.explicitConsentMode();
```
Comment (marcellustavares) [inline modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java:191]: @marcosapmf let's check here also for the `enabled` flag. Even if `explicitConsentMode` is true, it should take any effect unless the enabled checkbox is also true.

Comment (marcellustavares) [reply modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java:191]: Screenshot 2024-03-12 at 3 22 47 PM

In this case above, explicitConsentMode should be treated as false.

**Review by marcellustavares — CHANGES_REQUESTED** [review]

File: modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java (hunk 1 above)
Comment (marcellustavares) [reply modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java:191]: Please replace it to 

`return cookiesPreferenceHandlingConfiguration.enabled() && cookiesPreferenceHandlingConfiguration.explicitConsentMode();`

**Comment by marcosapmf** [comment]
ci:test:relevant

**Review by marcellustavares — APPROVED** [review]

**Comment by marcellustavares** [comment]
ci:forward:force

**Comment by ViniLopes20** [comment]
ci:test:stable

**Comment by interaminense** [comment]
Resend: https://github.com/liferay-ac/liferay-portal/pull/1271

**Pull request closed without merging**